tools/gxc/src/gxc.cpp

```diff
@@ -134,8 +136,12 @@ int runBuild(const CommandLineOptions& options)
         }
         try
         {
-            FileStream fs(manifestEntry.path, StreamFlags::read);
-            auto img = Image::fromPng(fs);
+            if (imageCache.count(manifestEntry.path) == 0)
```

Comment:
Using `.find()` allows you to both check it exists and returns the iterator/value if it does avoiding multiple lookups.

Reply:
done